pkg/virt-handler/isolation/detector_test.go

```diff
@@ -143,6 +143,23 @@ var _ = Describe("Isolation Detector", func() {
 			Expect(result.NetNamespace()).To(Equal(fmt.Sprintf("/proc/%d/ns/net", os.Getpid())))
 		})
 	})
+
+	Context("VM with VFIO device", func() {
+		It("should adjust QEMU process MEMLOCK rlimits", func() {
```

Comment:
I think that except `findIsolatedQemuProcess`, the other functions are hard to test.

I do not have a good solution at this point, but we should probably look in a following refactor on how to resolve this nicer.
If other reviewers have a better idea, maybe we can do something else.

Reply:
I just realized how wrong this is.. sorry.
I removed this awkward test with testing `findIsolatedQemuProcess` which is what make sense to test
And considering the current design is what possible to test.